data/categories/sg_sporting_goods.yml

```diff
@@ -6935,6 +6937,167 @@
   attributes:
   - color
   - pattern
+- id: sg-4-1-10
+  name: Wingfoils
+  children:
+  - sg-4-1-10-1
+  - sg-4-1-10-2
+  - sg-4-1-10-3
+  - sg-4-1-10-4
+  - sg-4-1-10-5
+  attributes: []
+- id: sg-4-1-10-1
+  name: Wingfoil Wings
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
+  - recommended_skill_level
+- id: sg-4-1-10-2
+  name: Wingfoil Foils
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
+  - recommended_skill_level
+- id: sg-4-1-10-3
+  name: Wingfoil Parts
+  children:
+  - sg-4-1-10-3-1
+  - sg-4-1-10-3-2
+  attributes: []
+- id: sg-4-1-10-3-1
+  name: Wing Parts
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
+- id: sg-4-1-10-3-2
+  name: Foil Parts
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
+- id: sg-4-1-10-4
+  name: Wingfoil Accessories
+  children:
+  - sg-4-1-10-4-1
+  - sg-4-1-10-4-2
+  attributes: []
+- id: sg-4-1-10-4-1
+  name: Wingfoil Carry Bags & Covers
+  children: []
+  attributes:
+  - bag/case_features
+  - bag/case_material
+  - color
+  - pattern
+- id: sg-4-1-10-4-2
+  name: Wingfoil Protective Gear
+  children: []
+  attributes: []
```

Comment:
If protective gear isn't substantially different across various water sports, we could create a `Water Sport Protective Gear` category under `Boating and Water Sport Apparel`, and move `Water Sports Helmets` inside it, one level deeper.

![image](https://github.com/user-attachments/assets/0ae66fb9-04d9-4033-a57c-ed4e1d43c09e)